payments-core/src/test/java/com/stripe/android/StripeKtxTest.kt

```diff
@@ -284,32 +284,20 @@ internal class StripeKtxTest {
         runTest {
             whenever(
                 mockApiRepository.confirmSetupIntent(any(), any(), any())
-            ).thenThrow(mock<AuthenticationException>())
+            ).thenReturn(Result.failure(mock<AuthenticationException>()))
 
             assertFailsWith<AuthenticationException> {
                 stripe.confirmSetupIntent(mock())
             }
         }
 
-    @Test
-    fun `When repository returns null then confirmSetupIntentSuspend should throw InvalidRequestException`(): Unit =
```

Comment:
Can the repository no longer return null? Did it ever return null in practice before?

Reply:
It can no longer return `null`. It _technically_ could return `null` before, but it’s not something that would happen in practice: we either return the object or we throw an exception.